client/extensions/woocommerce/components/form-dimensions-input/index.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormTextInput from 'components/forms/form-text-input';
+import FormTextInputWithAffixes from 'components/forms/form-text-input-with-affixes';
+
+function FormDimensionsInput( {
+	className,
+	dimensions,
+	unit,
+	onChange,
+	translate,
+} ) {
+	const classes = classNames( 'form-dimensions-input', className );
+
+	return (
+		<div className={ classes }>
+			<FormTextInput
+				name="length"
+				placeholder={ translate( 'L' ) }
+				type="number"
+				value={ dimensions && dimensions.length || '' }
+				onChange={ onChange }
+				className="form-dimensions-input__length"
+			/>
+			<FormTextInput
+				name="width"
+				placeholder={ translate( 'W' ) }
+				type="number"
+				value={ dimensions && dimensions.width || '' }
+				onChange={ onChange }
+				className="form-dimensions-input__width"
+			/>
+			<FormTextInputWithAffixes
+				name="height"
+				placeholder={ translate( 'H' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **8** times:
`translate( 'h' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).